(venv) jenny6199@notebook-6199:~/geekbrains/django_start_v2/src/geekshop$ python3 manage.py test authapp
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
db_profile UPDATE for <class 'mainapp.models.ProductCategory'>:
Installed 20 object(s) from 1 fixture(s)
No sequences found.
No sequences found.
No sequences found.
No sequences found.
.
----------------------------------------------------------------------
Ran 1 test in 1.989s

OK
Destroying test database for alias 'default'...
(venv) jenny6199@notebook-6199:~/geekbrains/django_start_v2/src/geekshop$ python3 manage.py dumpdata -e=contenttypes -e=auth -e=ordersapp -e=basketapp -e=sessions -o test_db.json

